pkg/controller/dynamic/dynamic_controller_integration_test.go

```diff
@@ -76,6 +76,11 @@ func init() {
 	flag.StringVar(&runTestsRegex, "run-tests", "", "run only the tests whose names match the given regex")
 	flag.StringVar(&skipTestsRegex, "skip-tests", "", "skip the tests whose names match the given regex, even those that match the run-tests regex")
 
+	// Only run tests associate with Alpha resources
+	flag.BoolVar(&runAlphaTests, "run-alpha-tests", false, "only run the tests associate with Alpha resources")
```

Comment:
nit: the two flags feel a bit conflicting. If we only want to support two mutually exclusive scenarios (running either alpha or beta tests), a single flag like `--test-group=[alpha|beta]` might be clearer.  
If we want to allow running both alpha and beta tests together in the future, then maybe something like `--run-alpha-tests` and `--run-beta-tests`? Let me know your thoughts!